src/pages/settings/Wallet/ExpensifyCardPage.tsx

```diff
@@ -92,6 +95,12 @@ function ExpensifyCardPage({
 
     useBeforeRemove(() => setIsValidateCodeActionModalVisible(false));
 
+    useEffect(() => {
+        return () => {
+            clearActivatedCardPin();
```

Comment:
Not sure if it makes a difference performance-wise - but should we check if the PIN is set before clearing it? Instead of always clearing it?